Dockerfile

```diff
@@ -12,13 +12,15 @@ RUN yum -y install epel-release \
 ARG OMERO_VERSION=latest
```

Comment:
If we're using the main release version (i.e. `5.3.2`) for tags of this repo, then I'm assuming:

 * `latest` images are considered worst-practice
 * on each OMERO release, we'll branch from `master` and bump this string to include the version (`5.3.3`)
 * for bug fixes before an OMERO release, we'll suffix `5.3.3-1`

Note: this doesn't yet give us a strategy for breaking changes in the image _itself_